src/main/java/io/specto/hoverfly/junit/core/RemoteHoverflyConfigBuilder.java

```diff
@@ -0,0 +1,79 @@
+package io.specto.hoverfly.junit.core;
+
+public class RemoteHoverflyConfigBuilder implements RemoteHoverflyConfig {
+
+    private static final int DEFAULT_PROXY_PORT = 8500;
+    private static final int DEFAULT_ADMIN_PORT = 8888;
+    private static final int DEFAULT_HTTPS_ADMIN_PORT = 443;
+
+    private String host;
+    private String scheme;
+    private String authToken;
+    private int adminPort = DEFAULT_ADMIN_PORT;
+    private int proxyPort = DEFAULT_PROXY_PORT;
+    private String sslCert; // file name relative to test resources folder
+    private String destination;
+    private boolean proxyLocalHost;
+
+
+    @Override
+    public RemoteHoverflyConfig withHttps() {
+        return this;
+    }
+
+    @Override
+    public RemoteHoverflyConfig withAuthHeader() {
+        return this;
+    }
+
+    @Override
+    public RemoteHoverflyConfig host(String host) {
+        this.host = host;
+        return this;
+    }
+
+    @Override
+    public RemoteHoverflyConfig withHttps(String sslCert) {
```

Comment:
This is the CA cert, not the one for the admin endpoint. You are right, we need to split this into two: 
`withHttpsAdminEndpoint`
`proxyCaCert`